### ISSUE TITLE
v2 doesn't work on ios9

### ISSUE THREAD

**Issue by daisymarie128** [goldfire/howler.js#449]
Just flagging I found that in v2.0.0-beta7, sounds would not play on ios9

tested on iphone5c ios 9.1 and iphone6+ ios9.2.

Sound wouldn't play 95% of the time (very weird edge cases when it would play a short snippet), even while calling play on touchend events. I also got no errors in the console. To fix this I went back to using v1.1.29 and everything works fine.

Cheers

**Comment by nhoizey** [comment]
I use v2.0.0-beta7 on iOS 9.2.1 (and previously 9.2) without issues

**Comment by goldfire** [comment]
The 2.0 tests also work on iOS 9 without any issues. Can you provide a test case including the audio file you are testing?

**Comment by daisymarie128** [comment]
Sorry I can't provide the audio file or a test case at the moment. Also I forgot to mention it worked in the chrome browser on ios9 but not safari.

I did test your examples on http://goldfirestudios.com/blog/104/howler.js-Modern-Web-Audio-Javascript-Library and that did work. The only other example on the web I could find which did the same thing was this http://www.williammalone.com/posts/ios-9-web-audio-api-safari-mobile-issue/ -- none of their examples that say it should work on ios9 do. if that's any help.

But as I mentioned I used the exact same code pretty much in v2 as v1 and after switching back to v1.1.29 and it worked fine. 

Sorry for the lack of detail
Cheers
